### PR TITLE
mock-tikv: update to reflect the change of tikv

### PR DESCRIPTION
**Pull request by tiancaiamao**
after PR https://github.com/pingcap/tidb/pull/1692, tipb.SelectResult use chunks instead of rows field in the protocol.

mock tikv still use rows, so the code is outdated and test cases cannot reflect the correct status.

this PR is an update, later we should clean up the unused code...

### CODE REVIEW

File: store/tikv/mock-tikv/cop_handler.go [hunk 1]
```diff
@@ -117,6 +117,49 @@ func (h *rpcHandler) handleCopRequest(req *coprocessor.Request) (*coprocessor.Re
...
+		RowsMeta: make([]tipb.RowMeta, len(rows)),
+	}
+
+	for i := 0; i < len(rows); i++ {
+		rowsMeta := &chunk.RowsMeta[i]
+		rowsMeta.Length = int64(len(rows[i].Data))
+		if handleBytes := rows[i].GetHandle(); len(handleBytes) > 0 {
+			_, datum, err := codec.DecodeOne(handleBytes)
+			if err != nil {
+				panic(err)
```
Comment (shenli) [inline store/tikv/mock-tikv/cop_handler.go]: Why not return error here?

**Comment by coocood** [comment]
@tiancaiamao Try to not use `*tipb.Row` as it will be removed later.

**Comment by tiancaiamao** [comment]
@coocood avoid using `*tipb.Row` here involve quite amount of change which will be revisit later.
I'd rather make remove `*tipb.Row` in another PR.
hold on, I'll do it tomorrow.

**Comment by tiancaiamao** [comment]
PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
@shenli @zimulala @hanfei1991 PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**